src/github.com/getlantern/eventual/eventual_test.go

```diff
@@ -14,6 +15,7 @@ const (
 )
 
 func TestSingle(t *testing.T) {
+	goroutines := runtime.NumGoroutine()
```

Comment:
Yes!  If you look at my PR, you'll see that I did this in with your detour tracker - https://github.com/getlantern/lantern/pull/3993/files#diff-dfac99737640cd22fdabf4fc7fb4312bR310

I'll refactor that into its own package and then use it from the eventual test.